my-code/whats-in/src/pages/Movie/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import iconBack from '../../images/icon-arrow-grey.svg'
+import logoImdb from '../../images/logo-imdb.svg'
+import logoRottenTomatoes from '../../images/logo-rotten-tomatoes.svg'
+import iconLike from '../../images/icon-heart-grey.svg'
+import iconLikeWhite from '../../images/icon-heart-white.svg'
+
+export default function Moviepage( {movie, isMovieLiked, handleLike, likedMovies} ) {
+    const navigation = useNavigate();
+
+    const handleNavigation = () => {
+        navigation(-1)
+    }
+
+    const checkRating = () => {
+        const rottenTomatoesRating = movie.Ratings.find((rating) => rating.Source === "Rotten Tomatoes");
+        return rottenTomatoesRating ? rottenTomatoesRating.Value : 'N/A'
+    }
+
+    const formatDetailsList = (info) => {
+        return info.split(', ').map((item) =>  <li>{item}</li>)
+    }
```

Comment:
this is not formatting the detailsList but rather rendering it.
you could either:
- keep this `formatDetailsList` but rewrite it to just return an array of strings: `info.split(', ').map((item) => item)`
- rename this to `renderDetailsList` (not recommended)
- create a component like `<Details />` that would receive a string and render a list (recommended)
- just do it all inline.

also, it's good practice to add a `key` to an element when rendering lists
```jsx

```